roles/etcd/files/etcd.yaml

```diff
@@ -32,8 +32,8 @@ spec:
        name: master-data
     livenessProbe:
       exec:
-      initialDelaySeconds: 15
-      timeoutSeconds: 10
+      initialDelaySeconds: 45
+    terminationGracePeriodSeconds: 0
```

Comment:
I don't think setting terminationGracePeriod to zero in etcd is a good idea - cc @smarterclayton